### DISCUSSION TITLE
Release graph visualization as seperate package

### DISCUSSION THREAD

**Discussion by LukasPoque** [AykutSarac/jsoncrack.com#42] [Ideas]
Love this project 🎉 Especially for complex JSON structures this is way better than a nested tree view.

Do you have any plans on releasing the graph visualization as a separate package? It could be helpful in other projects and beneficial for this package since bugs/problems could be fixed much easier by other collaborators.

**Comment by victorbrambati** [comment]
This project use [reaflow](https://github.com/reaviz/reaflow) to graph visualization.

**Reply by LukasPoque** [reply]
Yeah, I know that reaflow is used as graph visualization. Maybe I have been a little misunderstood. I thought of a package that includes the analysis of a JSON input and creating the needed graph nodes from it. So basically something everybody could use in their project by simply including the package and providing a state for the JSON string (or object).

**Comment by AykutSarac** [comment]
That's a good idea. I'm considering to release both JSON Editor and Visio graph as separate packages.

**Reply by LukasPoque** [reply]
Sounds really cool :) Any updates on this?

**Reply by AykutSarac** [reply]
Planning to release it with v2.0.0 but there's a long way to go. Hoping it to be a great major release.

**Reply by LukasPoque** [reply]
Any updates on this topic? I know this is not a fast implemented thing but IMO this could be beneficial for the whole project because it's getting more modular and flexible.

**Reply by AykutSarac** [reply]
VS Code just has been released, there is no ETA however I'm currently working on it but probably not going to be released soon.
